daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf2.proto

```diff
@@ -145,86 +145,53 @@ message Kind {
   }
 }
 
-// Builtin primitive types
-enum PrimType {
+// Builtin types
+enum BuiltinType {
   // Builtin type 'Unit'
   UNIT = 0;
-
   // Builtin type 'Bool'
   BOOL = 1;
-
   // Builtin type 'Int64'
   INT64 = 2;
-
-  // Builtin type for legacy 'Decimal'
-  // Alias for (Numeric 10)
-  // *available in version < 1.7*
-  DECIMAL = 3;
```

Comment:
Should we also remove the ADD_DECIMAL, SUB_DECIMAL, etc. builtins in the same PR? Or maybe not remove DECIMAL in this PR.

Reply:
Ideally I should not have remove DECIMAL here. 
I think it is not worthwhile to revert that now.